apps/web/src/components/content/sections/unified-section.tsx

```
@@ -92,12 +96,85 @@ function downloadTextFile(filename: string, content: string) {
+/**
+ * TrustedHTML - Safely renders HTML content with XSS protection
+ * Sanitizes HTML using DOMPurify before rendering
+ */
+function TrustedHTML({ html, className }: { className?: string; html: string }) {
+  if (!html || typeof html !== 'string') {
+    return <div className={className} />;
+  }
+
+  // Sanitize HTML to prevent XSS
+  // During SSR, render unsanitized (will be sanitized on client)
+  const [safeHtml, setSafeHtml] = useState<string>(
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)